### PR TITLE
[operator-trivy] report-updater

### PR DESCRIPTION
**Pull request by miklezzzz**
## Description
Match CVE to BDU vulnerabilities.
If BDU exists - replace vulnerability ID with BDU
Overwrite link in the Grafana with fstek url, if we have BDU vulnerability
Close https://github.com/deckhouse/deckhouse/issues/5555

## Why do we need it, and what problem does it solve?
Use BDU vulnerabilities instead of CVE if flag is enabled

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [ ] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: operator-trivy
type: feature
summary: Add a flag to use BDU vulnerabilities instead of CVE in the vulnerability reports.
```

### CODE REVIEW

**Review by yalosev — DISMISSED** [review]

File: ee/modules/500-operator-trivy/openapi/config-values.yaml [hunk 1]
```diff
@@ -1,5 +1,13 @@
 type: object
 properties:
+  enableReportUpdater:
```
Comment (nabokihms) [inline ee/modules/500-operator-trivy/openapi/config-values.yaml:3]: Converting to what? Let's describe not the action but the result.

Example:
Action -> Enable report updater
Result -> Add links to BDU to all reports

File: ee/modules/500-operator-trivy/images/report-updater/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+FROM $BASE_GOLANG_20_ALPINE as artifact
+WORKDIR /src/report-updater
+COPY main.go go.mod go.sum /src/report-updater/
+COPY vulndb /src/report-updater/vulndb/
+COPY web /src/report-updater/web/
+
+RUN GOOS=linux GOARCH=amd64 CGO_ENABLED=0 go test ./...
+RUN GOOS=linux GOARCH=amd64 CGO_ENABLED=0 go build -ldflags="-s -w" -o report-updater main.go
+
+FROM $BASE_ALPINE
```
Comment (nabokihms) [inline ee/modules/500-operator-trivy/images/report-updater/Dockerfile:12]: Shouldn't it be based on DISTROLESS?

**Review by nabokihms — CHANGES_REQUESTED** [review]

File: ee/modules/500-operator-trivy/images/report-updater/Dockerfile (hunk 2 above)
Comment (yalosev) [reply ee/modules/500-operator-trivy/images/report-updater/Dockerfile]: Done

File: ee/modules/500-operator-trivy/openapi/config-values.yaml (hunk 1 above)
Comment (yalosev) [reply ee/modules/500-operator-trivy/openapi/config-values.yaml]: Renamed to `linkCVEtoBDU`

**Review by nabokihms — APPROVED** [review]

**Pull request merged**